kola/tests/coretest/core.go

```diff
@@ -254,13 +319,23 @@ func TestInstalledUpdateEngineRsaKeys() error {
 }
 
 func TestServicesActive() error {
-	//t.Parallel()
-	units := []string{
+	return servicesActive([]string{
 		"multi-user.target",
 		"docker.socket",
 		"systemd-timesyncd.service",
 		"update-engine.service",
-	}
+	})
+}
+
+func TestServicesActiveRHCOS() error {
+	return servicesActive([]string{
+		"multi-user.target",
+		"docker.service",
```

Comment:
This is currently active for now, but not sure if there's a better choice of service.

Reply:
With openshift/redhat-release-cores#7, we could switch this to `crio.service`?

Reply:
The `rdgo` sync is currently broken so that change isn't propogating to the images.